src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,61 @@
 package edu.kis.vh.nursery;
 
 public class DefaultCountingOutRhymer {
+//TODO-encapsulation methods
 
-    private int[] numbers = new int[12];
+    private static final int CAPACITY = 12;
+    private static final int MAX_INDEX = 11;
```

Comment:
This constant is related to CAPACITY. Changing only one leads to an error (inconsistency).